website/content/docs/job-specification/check.mdx

```diff
@@ -112,12 +112,12 @@ job "example" {
 - `path` `(string: <varies>)` - Specifies the path of the HTTP endpoint which
   will be queried to observe the health of a service. Nomad will automatically
   add the IP of the service and the port, so this is just the relative URL to
-  the health check endpoint. This is required for http-based health checks.
+  the health check endpoint. This is required for HTTP-based health checks.
 
 - `expose` `(bool: false)` - Specifies whether an [Expose Path](/nomad/docs/job-specification/expose#path-parameters)
   should be automatically generated for this check. Only compatible with
   Connect-enabled task-group services using the default Connect proxy. If set, check
-  [`type`][type] must be `http` or `grpc`, and check `name` must be set.
+  [`type`](#type) must be `http` or `grpc`, and check `name` must be set.
```

Comment:
this one here is the broken link yeah?

Reply:
Yep. Since it is on the same page, we can just link to the ID rather than having to make an additional reference link at the bottom.